essay.tex

```diff
@@ -0,0 +1,50 @@
+
+\section{Importance of virtual tools}
+One could argue that virtual tools are not essential. They are not required at all if, say, a team is fine at their current state of productivity. Why change what is not broken? Although, while it may temporarily tamper with progress, learning to cope and adapt to using virtual tools along with the Agile methodology can vastly improve time-boxing which thereafter positively affects team productivity, which is why it is an attractive approach in the game development industry. Even if it is minimal, simply communicating for a maximum of five minutes with colleagues about current tasks and barriers is enough to provide information to the rest of the group, which they in turn could lend some assistance in if needed. This has a positive effect in the long run, as it develops a sense of positivity and reassurance amongst group members, resulting in elevated levels of team productivity, which consecutively decreases the potential problems that could arise in terms of time-boxing such as not reaching the specified date.
+
+\section{Different personality traits}
+Personality traits have been attractive aspects of discussions in the computing field, the two notable ones being introvertism and extrovertism. The Agile approach is known to involve continuous communication between teams which could potentially take up much of the day, something that ultimately might not seem as attractive among introverts. Indeed, the Agile methodology has brought upon a much-needed touch both in computing and business areas where products can be delivered in an interactive and flexible manner - in contrast to the waterfall methodology -, however there is no doubt that it favours extroverted personalities. Being able to engage in quick but elaborate meetings, demonstrate incomplete work, tolerate critique and generally unmask one's self is crucial in adapting to the Agile methodology. The whole process forces extroverted performance and behaviour out of introverts. Of course, this is harmless and tolerable for a small amount of time, although the problem gradually intensifies when having to sustain the particular "act", which as expected can lead to exhaustion and eventually decrease team productivity.
+
```

Comment:
Will refine the personalities section a bit to fit its original intentions more (explain the various effects virtual tools evoke on different personality traits).